src/ChoiceFieldset/ChoiceFieldLabel.tsx

```diff
@@ -0,0 +1,14 @@
+import React, {useContext} from 'react'
+import ChoiceFieldsetListContext from './ChoiceFieldsetListContext'
+
+const ChoiceFieldLabel: React.FC = ({children}) => {
+  const choiceFieldsetListContext = useContext(ChoiceFieldsetListContext)
+  if (choiceFieldsetListContext === null) {
+    throw new Error('ChoiceFieldsetListContext returned null')
+  }
+  const {fieldComponent: FieldComponent} = choiceFieldsetListContext
+
+  return <FieldComponent.Label>{children}</FieldComponent.Label>
```

Comment:
Potential scope-creep, so feel free to ignore in this PR... shall we add a cursor: pointer to this label so it's consistent with the input? As clicking it will trigger the associated input? Feels a bit strange it doesn't in the examples.

Reply:
I could go either way. I don't think labels typically have `cursor: pointer`, but I think it's a nice enhancement.